cmake/ETLInstallOmniBot.cmake

```diff
@@ -27,6 +27,6 @@ LEG_DOWNLOAD(
 
 message(STATUS "Adding Omni-Bot to installer scripts")
 install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/${MODNAME}/omni-bot/"
-	DESTINATION "${INSTALL_DEFAULT_MODDIR}/${MODNAME}/omni-bot"
-	DIRECTORY_PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE GROUP_READ WORLD_READ
+	DESTINATION "/var/lib/etlegacy/omni-bot"
```

Comment:
This breaks our standalone shipped package.

Reply:
Its purpose was to ease the use of omni-bot with a linux dedicated server. /var is rw and /usr is read-only for the server dedicated user in my case. 
An other variable  as ${INSTALL_DEFAULT_MODDIR} would be helpful.

Reply:
I am not sure what you are trying to solve here.
The shipped omnibot works fine when it isn't moved outside of `INSTALL_DEFAULT_MODDIR`.
If you want to sue another package that installs OB globally in your system path (with pre-compiled packages, like the `etlegacy-omnibot ` package in Arch Linux AUR), you surely don't need ETL to erase these packages again.

Requesting another variable does not make sense to me.